docker/README.md

```diff
@@ -24,7 +24,7 @@ docker run -it --entrypoint expand-data-corpus.py opensearchstaging/opensearch-b
 # Building a Copy of OpenSearch Benchmark Image
 1. Git clone OpenSearch Benchmark Github repository
 2. Make changes locally
-3. Run `docker build -t <docker image name>:<optional tag> -f ./docker/Dockerfile .`
+3. Run `docker build --build-arg VERSION=<PYPI VERSION> -t <docker image name>:<optional tag> -f ./docker/Dockerfile .`. Ensure that build arg VERSION takes in an existing pypi version of OSB.
```

Comment:
A reference to the scenario where `Dockerfile-development` would be useful will help the reader understand why it is provided and which option to choose.